### DISCUSSION TITLE
Why does react-dropzone absolutely need an input tag ?

### DISCUSSION THREAD

**Discussion by AlanBlanchet** [react-dropzone/react-dropzone#1236] [General]
Hello community,

I have a project and if I don't add an input field like so :
```react
<input {...getInputProps()} />
```

Then the "open" function simply doesn't work.
Is it something normal ?

Here is a link to a [codesandbox](https://codesandbox.io/s/beautiful-worker-0ylzv7?file=/src/App.js:640-670) to see my code in action.

Thanks in advance.

**Comment by rolandjitsu** [comment]
@AlanBlanchet  yes, that is expected. The input is needed because there is no other way today, except using https://developer.mozilla.org/en-US/docs/Web/API/Window/showOpenFilePicker, to trigger the file selection window to open. 

If you check the shared link, in latest Chrome, Edge and Opera, you won't need the input, but the rest of the browsers will need it.
